bookmarks_recreate/README.md

```diff
@@ -0,0 +1,9 @@
+Bookmarks recreate
+==================
+
+Save and restore bookmarks to and from a file.
+
+	To save current bookmarks to a file:
+	ubooksave <filename>
+	To restore bookmarks from a file:
+	ubookrestore <filename> 
```

Comment:
Maybe add some formatting, for instance:
```
* To save the current bookmarks to a file: `ubooksave <filename>`
* To restore...
```

Reply:
Done